contents/articles/2017-08_pupil_mobile/index.md

```diff
@@ -0,0 +1,23 @@
+--- 
+title: Introducing Pupil Mobile
+date: Thu Aug 05 2017 14:36:00 GMT+0700 (ICT) 
+author: Pupil Dev Team 
+subtitle: "Pupil Mobile is a companion app to Pupil Capture and Pupil Service. The application enables you to connect your Pupil eye tracking headset to your Android device via USBC. Preview video and other sensor streams and stream video data over a WiFi..."
+featured_img: "../../../../media/images/blog/pupil_mobile.jpg"
+featured_img_thumb: "../../../../media/images/blog/thumb/pupil_mobile.jpg"
+---
+
+We are excited to introduce Pupil Mobile! A companion app to Pupil Capture and Pupil Service, currently in public alpha at [Google Play Store](https://play.google.com/store/apps/details?id=com.pupillabs.pupilmobile)! 
+
+<div class="Feature-video-container-16by9">
+  <iframe class="Feature-video u-padTop--2" src="https://www.youtube.com/embed/atxUvyM0Sf8?rel=0" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe>
+</div>
+<div class="small u-padBottom--2">Introducing Pupil Mobile</div>
+
+The application enables you to connect your Pupil eye tracking headset to your Android device via USBC. You can preview video and other sensor streams on the Android device and stream video data over a WiFi network to other computers (clients) running Pupil Capture. And also save video from your Pupil eye tracking headset other sensor data locally on your Android device.
```

Comment:
The Pupil Mobile app enables you to connect your Pupil eye tracking headset to your Android device (Nexus 5x or Nexus 6p) via USBC. You can preview video and other sensor streams on your Android device and stream video data over a WiFi network to other computers (clients) running Pupil Capture. 

You can also use Pupil Mobile to save video from your Pupil eye tracking headset other sensor data locally on your Android device.